examples/cms-sanity/lib/sanity.js

```diff
@@ -1,7 +1,5 @@
-import {
-  createImageUrlBuilder,
-  createPreviewSubscriptionHook,
-} from 'next-sanity'
+import createImageUrlBuilder from '@sanity/image-url'
```

Comment:
Is there a reason this was removed from the next-sanity package?

Reply:
Similar as https://github.com/vercel/next.js/pull/34898#discussion_r816949741 I assume:

https://github.com/sanity-io/next-sanity#createimageurlbuilder-is-removed

Reply:
Yes, we're working on a better way to handle images that isn't using the image url builder pattern.
And we won't use this image builder utility.